src/core/simulation.jl

```diff
@@ -1381,3 +1384,30 @@ function deserialize_model(
         cd(orig)
     end
 end
+
+function serialize_status(sim::Simulation)
+    data = Dict("run_status" => string(get_simulation_status(sim)))
```

Comment:
can't we just store a string the store attributes instead of adding an additional file to the output? seems strange to store a json with just the status.